spec/views/users/create.html.erb_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
The default specs should also not be included. Even if you generate them all at once you can pick the files (or even the lines) you want to add to the commit, including them only when they have meaningful content.